docs/changes.txt

```diff
@@ -239,33 +239,215 @@ Changes in behaviour which may result in build errors
+- Add wxWebView::EnableBrowserAcceleratorKeys() (Tobias Taschner, #23070)
+- Add wxWebView::EnablePersistentStorage() (Tobias Taschner, #24405).
+- Add wxWebView::GetNativeConfiguration() (Tobias Taschner, #22781).
+- Add wxWebView::SetProxy() (#23854).
+- Add wxWebViewConfiguration (Tobias Taschner, #23374).
+- Allow resizing row and column labels in wxGrid (Dietmar Schwertberger, #24362).
+- Allow setting fonts in wxAuiTabArt to work (#24829).
+- Allow specifying size of wxTextEntryDialog (Blake-Madden, #24467).
+- Allow using wxVListBox wxXRC_NO_SUBCLASSING (Bill Su, #25144).
+- Derive wxSearchCtrl from wxTextEntry in all ports (Martin Corino, #23686).
+- Fix default background of RLE bitmaps (Brian Nixon, #23599).
+- Fix handling of sizers inside <windowlabel> XRC tag.
+- Fix loading some bitmaps using BI_BITFIELDS (Brian Nixon, #23601).
+- Fix print preview in high DPI (taler21, #24666).
+- Fix wxAUI dock art appearance after DPI change (Maarten Bent, #23420).
+- Fix wxFontList::FindOrCreateFont() for non-integral sizes (PB).
+- Fix wxGrid::GetDropTarget() (Dan Allen, #24448)
+- Fix wxNUM_VAL_ZERO_AS_BLANK behaviour on focus loss (#24859).
+- Fix wxPGVIterator used in wxPropertyGridManager (#23273).
+- Fix wxTextCtrl proof checking functions return values (mcorino, #23832).
+- Implement client data support in wxAuiToolBar (Miguel Gimenez, #23271).
+- Improve generic and MSW "About" dialog appearance (#23196).
+- Improve item sorting in generic wx{File,Dir}Ctrl (PB, #23962).
+- Improve wxImage resizing quality (Alex Shvartzkop, #25252).
+- Make date/time picker controls locale-aware (Ulrich Telle, #23965).
+- Make wxEL_ALLOW_NEW without wxEL_ALLOW_EDIT work (Martin Corino, #24099).
+- Make wxGenericValidator more flexible (Bill Su, #24134).
+- Make wxGrid::ProcessTableMessage() more convenient to use (Joachim Wiesemann).
+- Provide reason for wxNotificationMessage dismissal (Lauri Nurmi, #24340).
+- Restore non-live-resize in wxAUI and wxSplitterWindow (AliKet, #24193).
+- Save/restore wxAuiNotebook layout (#24950).
+- Several fixes to scrolling in wxVListBox (Bill Su, #24278).
+- Support high DPI bitmaps in wxBusyInfo (#23813).
+- Support specifying dark mode colours in XRC (#23571).
+- Support wxCHK_UNDETERMINED in wxUpdateUIEvent (Bill Su, #24764).
+- Update Scintilla to 5.0 and Lexilla to 5.3 (Maarten Bent, #23117, #24369).
+- Update nanosvg to fix possible crashes (#24213).
+- Use CSS colour names (#23518).
+- Use new default flat tab art provider in wxAUI (#25316).
+- Use thousands separators in page numbers when printing (Blake-Madden, #25320).
+
+wxGTK:
+
+- Add support for building with libwebkit2gtk-4.1 (Scott Talbert, #23633).
+- Add support for wxRB_SINGLE (Stefan Hansson, #23652).
+- Add wxTextCtrl::GTKSetPangoMarkup() (Blake-Madden, #24912)
+- Add wxWindow::GTKGetWin32Handle() (Ryan Ogurek, #24675).
+- Allow copying text in wxMessageDialog under GTK (Ian McInerney, #23039)
```

Comment:
This is already in 3.2 in https://github.com/vadz/wxWidgets/commit/e2470b2c8ba8d3cf39b651c3ed5f4a40f6341dc5, so it shouldn't need to be listed here (it is in the changelog for 3.2.2).

Reply:
Removed, thanks!